### PR TITLE
remove support for country and commodity params

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -11,38 +11,15 @@ class ApplicationController < ActionController::Base
...
-    country_name = params[:country]
-    country_id = params[:country_id]
-    commodity_name = params[:commodity]
-    commodity_id = params[:commodity_id]
 
-    if (not context_id.nil?)
+    if context_id.present?
       @context = Context.find(context_id)
 
       raise ActionController::ParameterMissing, 'The provided context could not be found' if @context.nil?
```
Comment (agnessa) [inline app/controllers/application_controller.rb]: actually this line will never execute - if context is not found, find will throw an exception

File: spec/controllers/download_controller_spec.rb [hunk 2]
```diff
@@ -3,17 +3,20 @@
 RSpec.describe DownloadController, type: :controller do
   include_context "two flows"
   describe "GET index" do
+    before(:all) do
+      context = Context.first
```
Comment (agnessa) [inline spec/controllers/download_controller_spec.rb]: this is probably not necessary (same in the other controller spec) - context variable is defined in the included rspec context and so in this situational context (!) probably no need to redefine it ;) if it ends up being nil maybe there's some other problem

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -11,38 +11,15 @@ class ApplicationController < ActionController::Base
...
-      raise ActionController::ParameterMissing, 'Required country missing' if country_name.nil? and country_id.nil?
-      raise ActionController::ParameterMissing, 'Required commodity missing' if commodity_name.nil? and commodity_id.nil?
-
-      if (not country_id.nil? and not commodity_id.nil?)
-        @context = Context
-                       .where('commodity_id = ?', commodity_id.to_i)
-                       .where('country_id = ?', country_id.to_i)
-                       .first
-      else
-        @context = Context
```
Comment (agnessa) [inline app/controllers/application_controller.rb]: love to see it gone, really hated this `ILIKE` bit :)

**Comment by tiagojsag** [comment]
Done, thanks for the feedback

**Pull request merged**